server/plugin.go

```diff
@@ -219,6 +271,7 @@ func (p *Plugin) handleList(w http.ResponseWriter, r *http.Request) {
 		lt := time.Unix(lastReminderAt/1000, 0).In(timezone)
 		if nt.Sub(lt).Hours() >= 1 && (nt.Day() != lt.Day() || nt.Month() != lt.Month() || nt.Year() != lt.Year()) {
 			p.PostBotDM(userID, "Daily Reminder:\n\n"+issuesListToString(issues))
+			p.trackDailySummary(userID)
```

Comment:
What is the value we get from tracking this event?

Reply:
Understanding how many daily summaries are being sent. May allow us to see errors in the code (more than one daily summary per user), or see how active are the users of the plugin.

Reply:
I think it makes sense to add this because we now have the user setting to disable the reminders.